guides/covariates_tfms.md

```diff
@@ -0,0 +1,289 @@
+# In-depth look at Covariates in Torch Forecasting Models
```

Comment:
How about renaming the document to `torch_forecasting_models.md` ?
You are actually covering more than just covariates (which is great :) )